mirror-service/src/main/java/org/acme/osv/OsvAnalyzer.java

```diff
@@ -0,0 +1,72 @@
+package org.acme.osv;
+
+import kong.unirest.json.JSONObject;
+import org.acme.client.OsvClient;
+import org.acme.model.OsvAdvisory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+import static org.acme.util.FileUtil.deleteFileAndDir;
+
+@ApplicationScoped
+public class OsvAnalyzer {
```

Comment:
This class should be renamed to be more reflective of what it does. It is not analyzing anything, so `Analyzer` may not be the right name.